tests/benchmark/generate_csv.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import argparse
+import sys
+import json
+import csv
+
+
+def generate_csv(args):
```

Comment:
Strongly prefer you to pass in each individual argument used from the command line args, in this case, ```def generate_csv(filename, tag)```.

This allows you to more completely document the inputs and outputs for the function, and makes it much easier to test.

(Please apply this comment to every function that accepts "args" in this file.)